ddl/column.go

```diff
@@ -880,6 +880,7 @@ func (w *worker) doModifyColumnTypeWithData(
 			return ver, errors.Trace(err)
 		}
 
+		failpoint.Inject("mockDelayInModifyColumnTypeWithData", func() {})
```

Comment:
Don't you need to do anything in this failpoint function? Then why do you need to add this failpoint.

Reply:
I will enable this failpoint with "pause" and do some verification on TiFlash. But the test scripts are defined in another repo.

Reply:
If so, please add some comments here?

Reply:
addressed.